ovos_PHAL_plugin_oauth/__init__.py

```diff
@@ -129,7 +147,7 @@ def handle_start_oauth(self, message):
         )
 
     @app.route("/auth/callback/<munged_id>", methods=['GET'])
-    def oauth_callback(self, munged_id):
+    def oauth_callback(munged_id):
```

Comment:
since the reference to `self` is not used can this method be moved to outside the plugin class maybe? keep it together with the app object since its the only method decorated with it